### PR TITLE
Update adapter.coffee

### PR DESCRIPTION
**Pull request by chapmanc**
Change private chat behavior so that a private chat room can have multiple members and hubot can respond to everyone in the private chat. 

Instead of responding to msg.send_email  grab the display recipients and do a private message with all of them.

### CODE REVIEW

File: adapter.coffee [hunk 1]
```diff
@@ -92,7 +92,8 @@ decode = (s) ->
 
 room_for_message = (msg) ->
     if msg.type == 'private'
-        "pm-with:#{encode(msg.sender_email)}"
+        recipient_list =  (user.email for user in msg.display_recipient)
```
Comment (timabbott) [inline adapter.coffee]: Can you drop the double-space here?

**Comment by timabbott** [comment]
Cool, thanks for working on this @chapmanc!  Looks like a clear improvement.  To what extent have you tested this?

While you're fixing my small comment, can you also line-wrap the commit message and s/Changing/Change/ as per http://zulip.readthedocs.org/en/latest/code-style.html#version-control?  
Thanks!

**Comment by chapmanc** [comment]
Sounds good. Fixing now. Do you want me to  change the message above, or open a new pull request with a modified commit message?

**Comment by chapmanc** [comment]
Also have tested for the last three months with no issues.

**Comment by timabbott** [comment]
Great.  Definitely no need to create a new pull-request; you can just push updates to the branch.  I squashed the commits and merged this, thanks @chapmanc!

**Pull request closed without merging**